inference-engine/tests/functional/plugin/conformance/subgraphs_dumper/src/op_cloner.cpp

```diff
@@ -0,0 +1,147 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op_cloner.hpp"
+#include "ngraph/opsets/opset6.hpp"
+#include "common_test_utils/data_utils.hpp"
+
+namespace SubgraphsDumper {
+namespace {
+const std::shared_ptr<ngraph::Node> clone(const std::shared_ptr<ngraph::Node> &node) {
+    ngraph::OutputVector op_inputs;
+    for (const auto &input : node->inputs()) {
+        if (ngraph::op::is_constant(input.get_source_output().get_node_shared_ptr())) {
+            op_inputs.push_back(input.get_source_output().get_node_shared_ptr()->clone_with_new_inputs({}));
+        } else {
+            op_inputs.push_back(std::make_shared<ngraph::op::Parameter>(input.get_element_type(),
+                                                                        input.get_source_output().get_shape()));
+        }
+    }
+    auto op_clone = node->clone_with_new_inputs(op_inputs);
+    return op_clone;
+}
+
+template<typename dType>
+std::shared_ptr<ngraph::op::Constant>
+copy_constant_with_randomization(const std::shared_ptr<ngraph::op::Constant> &const_node) {
+    std::vector<dType> data = const_node->cast_vector<dType>();
+    dType min = *std::min_element(data.begin(), data.end());
```

Comment:
1) Are you sure `data` is not empty?
2) https://en.cppreference.com/w/cpp/algorithm/minmax

Reply:
Thanks, good point. Fixed